### DISCUSSION TITLE
Importing large repo from Gitlab to Github fails

### DISCUSSION THREAD

**Discussion by iamvaibhavrai** [community/community#40595] [New to GitHub]
Hey everyone, I am trying to import a repo from Gitlab to Github with all the branches and commit history. I first tried using the Github import tool but it fails at the end with this error.
![image](https://user-images.githubusercontent.com/6650437/205217289-0c3cbb4a-9eb1-435c-a50d-4e18e66b141a.png)

I also tried to change the origin of the repo and push it to the GitHub repo from the terminal, but as the repo is larger than 2GB, I get the following error while running `git push --mirror github`

remote: fatal: pack exceeds maximum allowed size (2.00 GiB)
fatal: sha1 file '<stdout>' write error: Broken pipe MiB/s

Can someone help me with a workaround?

**Comment by radar-blip** [comment]
Probably your only option is [LFS](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage)

**Reply by EoFMimir** [reply]
Not working with LFS either
